distribution_circuits_sale/__manifest__.py

```diff
@@ -25,12 +25,14 @@
         'demo/demo.xml',
     ],
     'data': [
+        'data/mail_template.xml',
```

Comment:
website : https://www.coopiteasy.be and remove duplicate
increment version
add you name in the header of all the py files